llvm/docs/Contributing.rst

```diff
@@ -105,9 +105,16 @@ get the attention of potential reviewers by CC'ing them in a comment -- just
 A reviewer may request changes or ask questions during the review. If you are
 uncertain on how to provide test cases, documentation, etc., feel free to ask
 for guidance during the review. Please address the feedback and re-post an
-updated version of your patch. This cycle continues until all requests and comments
-have been addressed and a reviewer accepts the patch with a `Looks good to me` or `LGTM`.
-Once that is done the change can be committed. If you do not have commit
+updated version of your patch. This cycle continues until all requests and
+comments have been addressed and the reviewer accepts the patch with a `Looks
+good to me` or `LGTM`. With multiple active reviewers (i.e. reviewers who left
```

Comment:
Should we s/LGTM/approval/ (as in the button in the github ui)? I think 'LGTM' might be the language coupled to some older iteration of our code review process that feels a bit out of place now